build/linux/installer/scripts/tomlparser-agent-config.rb

```diff
@@ -222,6 +233,9 @@ def populateSettingValuesFromConfigMap(parsedConfig)
   if @fbitTailMemBufLimitMBs > 0
     file.write("export FBIT_TAIL_MEM_BUF_LIMIT=#{@fbitTailMemBufLimitMBs}\n")
   end 
+  if @syslogMonitoringMaxEventRate > 0
+    file.write("export MONITORING_MAX_EVENT_RATE=#{@syslogMonitoringMaxEventRate}\n")
```

Comment:
This should be only set for daemonset not the replicaset.

Reply:
will using ENV["CONTROLLER_TYPE"] for checking container type suffice here?